examples/src/main/scala/org/apache/spark/examples/streaming/StatefulNetworkWordCount.scala

```diff
@@ -44,18 +44,6 @@ object StatefulNetworkWordCount {
 
     StreamingExamples.setStreamingLogLevels()
 
-    val updateFunc = (values: Seq[Int], state: Option[Int]) => {
```

Comment:
if we want to encourage people to move to the new API, it might be ok to not have the old one.